tensor2tensor/data_generators/translate_enid.py

```diff
@@ -0,0 +1,85 @@
+# coding=utf-8
+# Copyright 2018 The Tensor2Tensor Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data generators for En-Id translation."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# Dependency imports
+
+from tensor2tensor.data_generators import problem
+from tensor2tensor.data_generators import text_encoder
+from tensor2tensor.data_generators import translate
+from tensor2tensor.utils import registry
+
+# End-of-sentence marker.
+EOS = text_encoder.EOS_ID
+
+# IWSLT17 :
+# 	109335 sentences
+#	https://wit3.fbk.eu/mt.php?release=2017-01-more
+# PANL-BPPT :
+#	24024 sentences
+# 	http://www.panl10n.net/english/outputs/Indonesia/BPPT/0902/BPPTIndToEngCorpusHalfM.zip
+_ENID_TRAIN_DATASETS = [
+    [
+        "https://github.com/prasastoadi/parallel-corpora-en-id/raw/master/IWSLT17.train.en-id.tok.tgz",
```

Comment:
Hi @prasastoadi -- Thanks for the PR ! This is going to be a good addition.

Our Travis tests seem to be failing because of the long lines here:
```
************* Module tensor2tensor.data_generators.translate_enid
C0301: 36 : Line too long (89/80) [line-too-long]
C0301: 39 : Line too long (103/80) [line-too-long]
C0301: 43 : Line too long (109/80) [line-too-long]
C0301: 47 : Line too long (109/80) [line-too-long]
C0301: 51 : Line too long (109/80) [line-too-long]
C0301: 55 : Line too long (109/80) [line-too-long]
C0301: 65 : Line too long (113/80) [line-too-long]
C0301: 66 : Line too long (88/80) [line-too-long]
```

Can we extract common parts of the variable out, something like:

REPO="https://github.com/prasastoadi/parallel-corpora-en-id/raw/master/"

and then rewrite it as:
```
_ENID_TRAIN_DATASETS = [
    [
        REPO + "IWSLT17.train.en-id.tok.tgz",
        ("IWSLT17.train.en-id.tok.en", "IWSLT17.train.en-id.tok.id")
    ],
   ...
]
```

Same with `_ENID_TEST_DATASETS`